src/App.vue

```diff
@@ -411,54 +337,89 @@ export default {
 			this.newBatchLabel = newBatchlabel;
 
 			this.newBatchBtnDisabled[mountidx] = true;
-			let endpoint =
-				this.parsedMounts[mountidx].mount_urloptions.ip +
-				":" +
-				this.parsedMounts[mountidx].mount_urloptions.debug_api_port;
-			if (!endpoint.toLowerCase().startsWith("http")) {
-				endpoint = "http://" + endpoint;
-			}
-			const requestOptions = this.getRequestOptions(
-				this.parsedMounts[mountidx].mount_urloptions.user,
-				this.parsedMounts[mountidx].mount_urloptions.password
-			);
 			console.log(
 				"batch,depth=" +
-					this.newBatchAmounts[mountidx] +
-					"," +
-					this.newBatchDepths[mountidx]
+				this.newBatchAmounts[mountidx] +
+				"," +
+				this.newBatchDepths[mountidx]
 			);
 
 			newBatchlabel = [...this.newBatchLabel];
-			try {
-				this.beeClient = new BeeDebug(endpoint);
-
-				const newBatchId = await this.beeClient.createPostageBatch(
-					Number(this.newBatchAmounts[mountidx]),
-					Number(this.newBatchDepths[mountidx]),
-					requestOptions
-				);
+			const url = generateUrl("/apps/files_external_ethswarm/bee/createPostageBatch");
```

Comment:
This isn't quite going to work as intended. It isn't guaranteed that an app will be under `/apps/`. For example on our production instance you will find most apps under `/custom_apps/`. I can't remember the syntax or which function exactly, but it should be possible to generate a URL that accounts for that as well.

Reply:
Nvm, it should already be handled by Nextcloud with how it is now